### PR TITLE
Rename 50 shades of greys

### LINKED ISSUE #606: [Proposal] Rename greys to be more abstract
# 2. Pattern proposal/amendment

## Is this a proposal or an amendment?

- Amendment: Colours

## Context

At the moment we have the following grey colour variable names:

- light grey
- warm grey
- mid grey
- cool grey

Platform has:

- porcelain (light grey) #f7f7f7
- silk (warm grey) #cdcdcd
- graphite (mid grey) #666
- jet/inkstone (cool grey) - jet is #111111 and inkstone is #3b3b3b, there is no #333333 - I propose inkstone

The proposal is that Vanilla:

- rename the current greys to a naming system based on the strength of the grey with a number from (0-100), for example $color-grey-10, $color-grey-40, etc.
- maybe we can find an automated way to find out what is the closest percentage of the grey based on white/black spectrum
- any time we refer to the actual HEX values in the CSS, comment the real name of the colour, as we will use it internally

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done
Renamed colour variables based on WG discussion:

```
x-dark: black
dark: #333333
mid-dark: #666666
mid-light: #cdcdcd
light: #f7f7f7
x-light: white
```

## QA
- Check build still works
- Cross ref hex codes with variable names
- Check color settings file in vf.io

## Details

Fixes #606

### CODE REVIEW

**Comment by barrymcgee** [comment]
@anthonydillon Conflicts should now be resolved.

**Comment by anthonydillon** [comment]
Looks like there is some conflicts in your vanilla-framework.io branch. Could you rebase master to help with the QA?

**Comment by barrymcgee** [comment]
https://github.com/ubuntudesign/docs-vanilla-theme/pull/28

File: docs/settings/_color-settings.md [hunk 1]
```diff
@@ -21,27 +21,27 @@ You can define a brand colour (<code>$color-brand</code>) that can be used for c
...
 <tr>
-<td><code>$color-cool-grey</code></td>
+<td><code>$color-mid-dark</code></td>
 <td style="background-color: #333; color: #fff;">#333</td>
 </tr>
 <tr>
-<td><code>$color-light</code></td>
-<td style="background-color: #fff;">#fff</td>
+<td><code>$color-dark</code></td>
+<td style="background-color: #666;">#666</td>
```
Comment (anthonydillon) [inline docs/settings/_color-settings.md]: I think this would be better with `color: #fff;`?

**Pull request merged**